src/unfold/contrib/import_export/forms.py

```diff
@@ -1,15 +1,29 @@
-from import_export.forms import ImportExportFormBase as BaseImportExportFormBase
-from unfold.widgets import SELECT_CLASSES, UnfoldAdminFileFieldWidget
+from import_export.forms import ExportForm as ExportFormBase
+from import_export.forms import ImportForm as ImportFormBase
+from unfold.widgets import UnfoldAdminFileFieldWidget, UnfoldAdminSelectWidget
 
 
-class ImportForm(BaseImportExportFormBase):
+class ImportForm(ImportFormBase):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.fields["resource"].widget = UnfoldAdminFileFieldWidget()
-        self.fields["format"].widget.attrs["class"] = " ".join(SELECT_CLASSES)
+        resource_widget = self.fields["resource"].widget
+        if len(resource_widget.choices) > 0:
+            self.fields["resource"].widget = UnfoldAdminSelectWidget(
```

Comment:
Replaced to using widgets here since they're already part of the unfold project and don't have to rewrite the same logic.
Not sure if there was a proper way of using this widget but this worked instead of just `UnfoldAdminSelectWidget`